language/stdlib/modules/doc/ValidatorConfig.md

```diff
@@ -230,6 +230,7 @@
         <a href="Signer.md#0x0_Signer_address_of">Signer::address_of</a>(signer) == <a href="#0x0_ValidatorConfig_get_operator">get_operator</a>(validator_account),
         1101
     );
+    Transaction::assert(<a href="Signature.md#0x0_Signature_ed25519_validate_pubkey">Signature::ed25519_validate_pubkey</a>(<b>copy</b> consensus_pubkey), 1108);
     // TODO(valerini): verify the validity of new_config.consensus_pubkey and
     // the proof of posession
```

Comment:
nit: you may want to edit this comment, and possibly link it to a new GH issue if you don't plan to tackle posession right away

Reply:
I actually wanted to tackle possession in this pull-request as well...

Reply:
The proof-of-possession sparkled some interesting discussions, we'll tackle it later.